torchci/lib/utilization/fetchUtilization.test.ts

```diff
@@ -213,7 +213,7 @@ describe("Test flattenTS to flatten timestamp", () => {
 
     // assert log
     expect(logSpy).toHaveBeenCalledWith(
-      `Warning: Error parsing JSON:SyntaxError: Expected property name or '}' in JSON at position 1 for data string '{{}dsad}'`
+      `Warning: Error parsing JSON:SyntaxError: Expected property name or '}' in JSON at position 1 (line 1 column 2) for data string '{{}dsad}'`
```

Comment:
accedental change?

Reply:
no tried to fix, but the message is not correct, I can remove the change